enemies/bosses/scenarios.json

```diff
@@ -70,6 +70,81 @@
       "incomingDamage": [],
       "note": "Botwoon 2 can not hit Samus while she is in the bottom left corner of the room."
     },
+    {
+      "id": 3,
+      "name": "Reverse Botwoon",
+      "boss": "Botwoon 1",
+      "requires": [],
+      "bossDodgeRate": 50,
+      "excludedWeapons": [
+        "Missile",
+        "Charge+Wave",
+        "Charge+Ice+Wave",
+        "Charge+Wave+Spazer",
+        "Charge+Ice+Wave+Spazer",
+        "Charge+Wave+Plasma",
+        "Charge+Ice+Wave+Plasma"
+      ],
+      "damageWindows": [
+        {
+          "name": "Magic Pixel Beam Fight",
+          "windowPercent": 15,
+          "requires": [
+            "Charge"
+          ],
+          "note": [
+            "Requires standing on a certain 2 pixel wide position from the wall.",
+            "Use angle up or angle down to shoot through the wall.",
+            "This does not include Charge plus Wave."
+          ]
+        },
+        {
+          "name": "Plasma Shield",
+          "windowPercent": 80,
+          "requires": [
+            "Charge",
+            "Plasma",
+            "PowerBomb"
+          ],
+          "note": "Freely fire Plasma Special Beam Attacks while close to the wall."
+        },
+        {
+          "name": "Peek Supers",
+          "windowPercent": 0.25,
+          "requires": ["Super"],
+          "note": "Wait for the one pattern (bottom->right) where Botwoon's head passes through the dividing barrier briefly and fire a Super Missile.",
+          "devNote": "Loosely 6seconds per pattern, 2seconds for acid, 16 patterns, 1 good pattern, acid after every second pattern. ~1 super per minute"
+        }
+      ],
+      "incomingDamage": [
+        {
+          "name": "Close Acid",
+          "attack": "acid",
+          "frequencyFrames": 3600,
```

Comment:
Are these frequencyFrames correct? Here's an analysis for if I'm understanding how this works correctly:
- In the vanilla Botwoon 1 boss scenario, it looks like with no items, Samus takes a hit every 20 seconds. With Gravity+Screw, she can dodge all damage, and Morph can be used to mitigate some damage. (Just gravity: 1 hit per 28.5 s, just morph: 1 hit per 33 s).
- In this, morph and screw don't help. With no items, Samus takes a hit every 14 seconds. With Gravity it is reduced to every 33 s. (For the 13 super strat, Samus will still take 24 acid hits even with Gravity+Screw+Morph)